roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -23,6 +23,7 @@
   register: install_ovs_sysconfig
   notify:
   - restart openvswitch
+  - restart openvswitch pause
```

Comment:
And if you only want it to happen for containerized environments just add a when to `restart openvswitch pause`